tests/saitests/py3/sai_qos_tests.py

```diff
@@ -1411,39 +1399,27 @@ def runTest(self):
 
         # get a snapshot of counter values at recv and transmit ports
         # queue_counters value is not of our interest here
-        recv_counters_base, queue_counters = sai_thrift_read_port_counters(
-            self.client, port_list[self.src_port_id]
-        )
+        recv_counters_base, queue_counters = sai_thrift_read_port_counters(self.client, asic_type, port_list[self.src_port_id])
```

Comment:
@XuChen-MSFT `asic_type` => `self.asic_type`
sorry for not finding it yesterday